templates/post.php

```diff
@@ -1,19 +1,16 @@
 <?php
-$wwwlib = sprintf(
-    "//%s/%sresources/", 
-    htmlentities($_SERVER['SERVER_NAME']), 
-    htmlentities($this->data['baseurlpath'])
-);
+use SimpleSAML\Utils\HTTP;
 ?>
+
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
         "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 <head>
     <meta http-equiv="content-type" content="text/html; charset=utf-8" />
     <title>POST data</title>
-    <script type="text/javascript" src="<?php echo $wwwlib;?>post.js"></script>
+    <script type="text/javascript" src="<?php echo HTTP::getBaseURL();?>/resources/post.js"></script>
```

Comment:
Hi again!

Sounds like you are mixing the server's path and the URL. It doesn't matter if you used symlinks, aliases or any other mechanisms during installation. `$this->data['baseurlpath']` will contain the relative base path (of the URL) to the script calling the template. Check it [here](https://github.com/simplesamlphp/simplesamlphp/blob/master/templates/includes/header.php#L56) for example.